google/cloud/spanner/internal/database_admin_metadata_test.cc

```diff
@@ -71,10 +72,10 @@ TEST_F(DatabaseAdminMetadataTest, CreateDatabase) {
   DatabaseAdminMetadata stub(mock_);
   CompletionQueue cq;
   gsad::v1::CreateDatabaseRequest request;
-  request.set_parent(
+  request.set_parent(internal::UrlEncode(
```

Comment:
Did you mean to include these (old interface) Spanner admin changes?  They should be no-ops given that instance/database names have syntax restrictions.  We also don't do this in the (new) generated admin interfaces.

Reply:
> They should be no-ops

I don't think that's true. Any `/` changes to `%2F`.

> We also don't do this in the (new) generated admin interfaces.

If "this" means "URL encode the values", then we do do it for the new admin clients:

https://github.com/googleapis/google-cloud-cpp/blob/2759683797c3b0645b36f10fd1fdb094b38c6dc0/google/cloud/spanner/admin/internal/database_admin_metadata_decorator.cc#L49-L50

Reply:
That's not the value in the request proto though.

Reply:
Ah, ok. So "this" meant "URL encode the value in the request proto". Yeah, we shouldn't do that anywhere.

Reply:
Right.  (And ack on the `%2F`.  Thanks.)

Reply:
Yeah, I want to modify the shared testing matcher, and spanner admin apis are using that matcher, but the current code would not work with my changes,